lib/src/main/java/edu/stanford/futuredata/macrobase/analysis/classify/MultiMADClassifier.java

```diff
@@ -0,0 +1,292 @@
+package edu.stanford.futuredata.macrobase.analysis.classify;
+
+import edu.stanford.futuredata.macrobase.datamodel.DataFrame;
+import edu.stanford.futuredata.macrobase.operator.Transformer;
+
+import org.apache.commons.math3.special.Erf;
+import org.apache.commons.math3.stat.descriptive.rank.Percentile;
+
+import java.lang.Math;
+
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Random;
+import java.util.stream.IntStream;
+
+public class MultiMADClassifier implements Transformer {
```

Comment:
Might be helpful to add a comment here on how this classifier works, e.g. it classifies each input column independently, and adds a column of results for each input column etc.